sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOStorageReadTest.java

```diff
@@ -622,40 +657,6 @@ public void testStreamSourceSplit() throws Exception {
     assertThat(streamSource.split(0, options), containsInAnyOrder(streamSource));
   }
 
-  @Test
-  public void testStreamSourceGetMaxEndOffset() throws Exception {
-
-    BigQueryStorageStreamSource<TableRow> streamSource =
-        BigQueryStorageStreamSource.create(
-            ReadSession.getDefaultInstance(),
```

Comment:
Why were these tests were removed ?

Reply:
#getMaxEndOffset and #createSourceForSubrange are properties of OffsetBasedSource. I switched the BigQuery Storage source to be a BoundedSource, which does not have these properties.